lib/tasks/import.rake

```diff
@@ -0,0 +1,87 @@
+namespace :host_reports do
+  def config_report_help(host_id, reported_at, metrics, origin)
+    {
+      :host_id => host_id,
+      :reported_at => reported_at,
+      :metrics => metrics,
+      :origin => origin,
+    }
+  end
+
+  def keywords(status)
+    keywords_set = {}
+    keywords_set["PuppetFailed"] = true if status["failed"] == 1
+    keywords_set["PuppetFailedToRestart"] = true if status["failed_restarts"] == 1
+    keywords_set["PuppetCorrectiveChange"] = true if status["corrective_change"] == 1
+    keywords_set["PuppetSkipped"] = true if status["skipped"] == 1
+    keywords_set["PuppetRestarted"] = true if status["restarted"] == 1
+    keywords_set["PuppetScheduled"] = true if status["scheduled"] == 1
+    keywords_set["PuppetOutOfSync"] = true if status["out_of_sync"] == 1
+    keywords_set.keys.to_a rescue []
+  end
+
+  def summary(resources, change, nochange, failure)
+    {
+      :foreman => {
+        :change => change,
+        :nochange => nochange,
+        :failure => failure,
+      },
+      :native => resources,
+    }
+  end
+
+  def create_body(metrics, reported_at, status, logs, host, change, nochange, failure)
+    body = {
+      :migrated => "true",
+      :host => host,
+      :reported_at => reported_at,
+      :logs => logs,
+      :keywords => keywords(status),
+      :summary => summary(metrics["resources"], change, nochange, failure),
+    }
+    body.to_json
+  end
+
+  def metrics_values(metrics)
+    change = metrics.dig("changes", "total") || 0
+    failure = metrics.dig("events", "failure") || 0
+    total = metrics.dig("events", "total") || 0
+    nochange = total - change || 0
```

Comment:
Both `total` and `change` should already be set, there should be no reason for `|| 0`